### ISSUE TITLE
(UI) Show "Selected UTXOs before performing transaction"

### ISSUE THREAD

**Issue by amitx13** [joinmarket-webui/jam#773]
Subtask of: #689 
**Aim** : To give user an overview of the selected `UTXOs` before performing transaction 
![image](https://github.com/joinmarket-webui/jam/assets/112259776/50dd0b35-e477-420c-8e2d-13d848f3ea3d)
**Objective**: Probably adding an extra field, i.e Selected UTXOs and showing the selected utxos to the user
@theborakompanioni and @editwentyone, please suggest if u have something else to make it more intuitive

**Comment by editwentyone** [comment]
good idea, make it a collapsible list (known from your other task) so its not getting to long

**Comment by amitx13** [comment]
Yeah sure, a Figma draft would be really helpful. Could you please provide one?

**Comment by editwentyone** [comment]
https://www.figma.com/design/kfejZJFlwBywvLEnPEmJo1/JoinMarket-UI?node-id=6978-17415&t=CnX9tjp55Ds1Y2VB-11 

<img width="797" alt="image" src="https://github.com/joinmarket-webui/jam/assets/96262473/62db1df9-eed2-449e-a91e-5d01c8a65c44">

simpler, without collapsible list

**Comment by theborakompanioni** [comment]
> simpler, without collapsible list

Can the UTXO list component of the fidelity bond flow be reused?
(Also.. in your screenshot, you show addresses, not UTXOs)

**Comment by amitx13** [comment]
I assume you mean something similar to the UtxoSummary like this, right? 

![Screenshot from 2024-06-21 12-06-34](https://github.com/joinmarket-webui/jam/assets/112259776/bd5dea97-df89-4a01-9fcf-16bceda85d27)

But here, just the address and amount are visible.

**Comment by amitx13** [comment]
we can also make it collapsible, as suggested by @editwentyone earlier . Take a look https://www.figma.com/design/SKNgJSH1c35awZDt3PQIJO/Untitled?node-id=0-1&t=xzTcinoUXb3U6Vkc-1

![image](https://github.com/joinmarket-webui/jam/assets/112259776/e98d62cc-5273-42d0-8a71-ac695e35ba8f)

**Comment by amitx13** [comment]
Hey @editwentyone if you have a moment, can you please share some thoughts on this?

**Comment by amitx13** [comment]
we can also keep it simple something like this 
![image](https://github.com/joinmarket-webui/jam/assets/112259776/7f3c7e08-bed6-4d95-9fe9-344706f9840f)
what do you think @editwentyone  ?

**Comment by editwentyone** [comment]
suggestion: 
<img width="1339" alt="image" src="https://github.com/joinmarket-webui/jam/assets/96262473/3fbe9f74-f9f8-480d-87bf-fc67db467a6b">
https://www.figma.com/design/kfejZJFlwBywvLEnPEmJo1/JoinMarket-UI?node-id=6978-17415&t=coq81rwHJwF1sIyA-4
